src/views/admin/AdminHearings.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import Helmet from 'react-helmet';
+import {injectIntl, intlShape} from 'react-intl';
+import {connect} from 'react-redux';
+import {fetchHearingList} from 'actions';
+import HearingList from 'components/HearingList';
+import Col from 'react-bootstrap/lib/Col';
+import Row from 'react-bootstrap/lib/Row';
+import {Tab, Tabs} from 'react-bootstrap';
+
+class AdminHearings extends React.Component {
+  static fetchData(dispatch) {
+    const now = new Date().toISOString();
+    return Promise.all([
+      dispatch(fetchHearingList("publishedHearings", "/v1/hearing/", {published: "True", open_at_lte: now})),
+      dispatch(fetchHearingList("publishingQueueHearings", "/v1/hearing/", {published: "True", open_at_gt: now})),
+      dispatch(fetchHearingList("draftHearings", "/v1/hearing/", {published: "False"}))
+    ]);
+  }
+
+  componentDidMount() {
+    const {dispatch} = this.props;
+    AdminHearings.fetchData(dispatch);
+  }
+
+  render() {
+    const {formatMessage} = this.props.intl;
+    const {hearingLists} = this.props;
+
+    return (<div className="container">
```

Comment:
Put the `div` into its own line. Makes this look more like properly intended HTML.